### ISSUE TITLE
Source Generated Properties in View Model

### ISSUE THREAD

**Issue by mthode88** [CommunityToolkit/dotnet#324]

**Comment by Sergio0694** [comment]
I'm pretty sure this has been fixed a while back, don't remember the exact PR off the top of my head. @mthode88 are you using 8.0.0-Preview4?

**Comment by mthode88** [comment]
No, I have not been using the Preview. I am using the latest stable version 7.1.2. I tried searching for the issue first before putting in this issue and wasn't able to find it. If it already has been solved, can you point me to the correct issue so I can review? Also, any idea when 8.0.0 will be fully released and not in a preview mode?

**Comment by Sergio0694** [comment]
I don't recall the exact PR where I fixed this off the top of my head, you'll have to go through the file history if you want to know that. Anyway the source generators in 7.x.x are in preview and not supported for production, and they have several known bugs at this point. I would recommend not using them. If you want to use the source generators, do use the latest 8.0.0 preview. They've been rewritten from scratch and they're completely different now 🙂

> "any idea when 8.0.0 will be fully released and not in a preview mode"

No specific ETA, thinking a few weeks maybe, but we'll see 😄

Closing this as resolved, since this is fixed now.
